apps/modernization-ui/src/routes/AppRoutes.tsx

```diff
@@ -91,8 +92,9 @@ export const AppRoutes = () => {
                                 {config.features.pageBuilder.enabled && (
                                     <Route path="/page-builder" element={<PageBuilderContextProvider />}>
                                         <Route path="pages">
-                                            {pageLibraryRoutes(config.features.pageBuilder.page.library.enabled)}
-                                            {pageManagementRoutes(config.features.pageBuilder.page.management.enabled)}
+                                            <Route path="valueset-library" element={<ValuesetLibrary />} />
```

Comment:
Issue: Value set library should only be accessible from within a modal during the create question flow.